docker/Dockerfile

```diff
@@ -119,6 +124,13 @@ RUN apt-get install -y \
     gir1.2-gst-rtsp-server-1.0 && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-#setup entrypoint to run the demo application
-ENTRYPOINT ["/usr/bin/entrypoint.sh"]
+# Install GPIO CPP dependencies
+WORKDIR /opt
+RUN wget -q https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/snapshot/libgpiod-2.1.tar.gz && \
+    tar -xf libgpiod-2.1.tar.gz && \
+    cd libgpiod-2.1 && \
+    bash ./autogen.sh && make install && \
+    rm -rf libgpiod-2.1 libgpiod-2.1.tar.gz
```

Comment:
Can you try installing it via apt, something like below

sudo apt-get install libgpiod>=2.0

Reply:
tried that out, didnt work